_15thnight/models.py

```diff
@@ -194,7 +198,7 @@ def get_alerts(cls):
     def get_provider_alerts(cls, provider):
         alerts = Alert.query \
             .join(cls.providers_notified) \
-            .filter(ProviderNotified.provider_id==provider.id) \
+            .filter(ProviderNotified.provider_id == provider.id) \
```

Comment:
Inside filter statements, we do not space out the equality check. Please change these back.